erpnext/accounts/doctype/sales_invoice/sales_invoice.json

```diff
@@ -428,7 +428,6 @@
    "no_copy": 1,
    "options": "Sales Invoice",
    "print_hide": 1,
-   "read_only_depends_on": "eval:doc.is_return",
```

Comment:
Why this is removed?

Reply:
Hi, this was removed for:
1- When we make a new return invoice, if, for any reason we need change return invoice reference . The check box is_return is 1, then the  field against_return is read only.
2- For normal Invoices the against_return is hidden. 
3- On Submit invoices we can't modify anything.